faqs.rst

```diff
@@ -16,33 +16,33 @@ Background
 What is Flux Framework?
 =======================
 
-Flux is a flexible framework for resource management, built for your site. 
-The core framework here consists of a suite of projects, tools, and libraries which may be used to build site-custom resource managers for 
+Flux is a flexible framework for resource management, built for your site.
+The core framework here consists of a suite of projects, tools, and libraries which may be used to build site-custom resource managers for
```

Comment:
```suggestion
The core framework here consists of a suite of projects, tools, and libraries that may be used to build site-custom resource managers for
```

Reply:
Just added a commit to make the word change.  I'm sort of ambivalent on the pre-commit hook.  Could we table that for now?